src/emulator/auth/operations.ts

```diff
@@ -1726,7 +1726,11 @@ function generateJwt(
     // This field is only set for anonymous sign-in but not for any other
     // provider (such as email or Google) in production. Let's match that.
     provider_id: signInProvider === "anonymous" ? signInProvider : undefined,
-    auth_time: toUnixTimestamp(new Date()),
+    auth_time: user.lastLoginAt
```

Comment:
```suggestion
    auth_time: user.lastLoginAt != null
```

`user.lastLoginAt` may as well be the number zero if it happens at epoch. (This won't happen in real applications, but may happen during tests where we mock out the clock)